assets/css/reset.css

```diff
@@ -45,6 +49,19 @@ body {
   --alert-grey-bg: #f5f5f5;
   --orchid-shade: #da70d6;
   --light-orchid: #bb86fc;
+  --primary-color1: #2b0e64;
+  --secondary-color1: #a2ffb9;
+  --tertiary-color1: #5e17eb;
+}
+
+.darkmode {
+  --off-white: #32174d;
+  --secondary-color: #a2ffb9;
+  --primary-color1: #ffffff;
+  --primary-color: #2b0e64;
+  --secondary-color1: var(--primary-color);
+  --tertiary-color1: var(--primary-color1);
```

Comment:
Appreciation: You have used consistent values of color only Hex and rgba, also everywhere you have used rems for responsiveness. Great Work.